authentication-request-handler/src/main/java/org/wso2/carbon/identity/framework/extended/ExtendedAuthenticationRequestHandler.java

```diff
@@ -0,0 +1,98 @@
+package org.wso2.carbon.identity.framework.extended;
+
+import org.apache.commons.lang.StringUtils;
+import org.wso2.carbon.identity.application.authentication.framework.context.AuthenticationContext;
+import org.wso2.carbon.identity.application.authentication.framework.exception.FrameworkException;
+import org.wso2.carbon.identity.application.authentication.framework.handler.request.impl.DefaultAuthenticationRequestHandler;
+import org.wso2.carbon.identity.application.authentication.framework.model.AuthenticatedUser;
+import org.wso2.carbon.identity.application.authentication.framework.util.FrameworkConstants;
+import org.wso2.carbon.identity.application.common.model.ClaimMapping;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+public class ExtendedAuthenticationRequestHandler extends DefaultAuthenticationRequestHandler {
+
+    private static final String ROLE_CLAIM = "http://wso2.org/claims/role";
+    private static final String ROLE_TO_CHECK = "trip";
+
+    @Override
+    protected void concludeFlow(HttpServletRequest request,
+                                HttpServletResponse response,
+                                AuthenticationContext context) throws FrameworkException {
+
```

Comment:
Please remove the additional new line.

Reply:
Fixed with d3c4ba88995f4b268756fa96d439f275f112d370